software/util/reorg.py

```diff
@@ -6,67 +6,104 @@
 """
 
 from collections.abc import Sequence
-from rdflib import Graph, Namespace
+from rdflib import Graph, Namespace, RDF, RDFS, DCTERMS, URIRef
+import argparse
+import itertools
 import logging
 import sys
-import argparse
 
-def SchemaOrgGraph():
-  """Creates a graph with usual namespaces set as we want in schemaorg."""
-  g = Graph()
-  g.bind("dc", Namespace("http://purl.org/dc/terms/"), replace=True)
```

Comment:
Maybe add a TODO to have these in a central place.

Reply:
Done.